### PR TITLE
add explainer

### PR DESCRIPTION
**Pull request by cvazac**

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Server Timing Explained
+
+## What's All This About?
+Server Timing allows developers to monitor the performance of server-handling of requests of every resource that makes up a page by overloading Navigation (for the base HTML) and Resource Timing (for scripts, images, etc). This new API enables developers to have insight into performance bottlenecks of the `Request` block below, that is, after `requestStart` and before `responseStart`.
```
Comment (igrigorik) [inline explainer.md]: Nitpicking language, but perhaps..

"Server Timing allows developers to monitor the performance of server-handling of requests of every resource that makes up a page by extending Navigation Timing and Resource Timing APIs. The new attributes exposed by Server Timing enable developers to have insights.."

File: explainer.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ 
+![navigation timing image](./nav-timing.png)
+
+Adhering to a standard response header format gives developer tools the ability to visualize backend timers that the developer (or intermediate proxies) deem important. The Javascript API gives analytics vendors the ability to collect and beacon more detailed timing data about each request.
```
Comment (igrigorik) [inline explainer.md]: "to a response header format defined by this specification gives developers" -- more explicit that we're defining it, not someone else.

File: explainer.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Server Timing allows developers to monitor the performance of server-handling of requests of every resource that makes up a page by overloading Navigation (for the base HTML) and Resource Timing (for scripts, images, etc). This new API enables developers to have insight into performance bottlenecks of the `Request` block below, that is, after `requestStart` and before `responseStart`.
+ 
+![navigation timing image](./nav-timing.png)
+
+Adhering to a standard response header format gives developer tools the ability to visualize backend timers that the developer (or intermediate proxies) deem important. The Javascript API gives analytics vendors the ability to collect and beacon more detailed timing data about each request. 
+
+### Goals
+The goal of this API is the standardization of the following:
+* a response-header format
+* the relationship between server timing data and other RUM APIs (Navigation and Resource Timing)
```
Comment (igrigorik) [inline explainer.md]: ... and Navigation and Resource Timing APIs?

File: explainer.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  for (const {name: url, serverTiming} of performance.getEntriesByType(entryType)) {
+    for (const {name, duration, description} of serverTiming) {
+      console.log('Server Timing entry =',
+        JSON.stringify({url, entryType, name, duration, description}, null, 2))
+    }
+  }
+}
+```
+
+## "_Controversial_" decisions
```
Comment (igrigorik) [inline explainer.md]: Do we need this at all? I propose we drop this section.

Comment (cvazac) [inline explainer.md]: changing title to "Considered alternatives" per proposed template [here](https://docs.google.com/document/d/1cJs7GkdQolqOHns9k6v1UjCUb_LqTFVjZM-kc3TbNGI/edit#heading=h.olht6c8vjwfl)

File: explainer.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        JSON.stringify({url, entryType, name, duration, description}, null, 2))
+    }
+  }
+}
+```
+
+## Considered alternatives
+
+### Explicitly named metadata parameters:
+There was [some discussion](https://github.com/w3c/server-timing/issues/12) about defining the `description` parameter as an explicitly named parameter. But as this API is intended to be as lightweight as possible, we've [suggested](https://github.com/w3c/server-timing/issues/22#issuecomment-317123400) [alternatives](https://github.com/w3c/server-timing/issues/12#issuecomment-317876891).
```
Comment (igrigorik) [inline explainer.md]: It's unclear what the conclusion is here. We suggested alternatives... and? 

"We investigated alternatives and decided to keep simple format, to align with original design goals of the API." or some such?

File: explainer.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+  }
+}
+```
+
+## Considered alternatives
+
+### Explicitly named metadata parameters:
+There was [some discussion](https://github.com/w3c/server-timing/issues/12) about defining the `description` parameter as an explicitly named parameter. But as this API is intended to be as lightweight as possible, we've [suggested](https://github.com/w3c/server-timing/issues/22#issuecomment-317123400) [alternatives](https://github.com/w3c/server-timing/issues/12#issuecomment-317876891). 
+
+### `duration` with no `startTime`? [wtf](https://github.com/w3c/server-timing/issues/10)?
```
Comment (igrigorik) [inline explainer.md]: Let's be careful with language.. :)

Also, isn't this already covered in non-goals? If you feel that we should keep this, I propose:
> ServerTiming records do not have a startTime?
>> See non-goals note on startTime and related discussion.

**Pull request merged**